server/command_create.go

```diff
@@ -170,29 +170,27 @@ func (p *Plugin) runCreateCommand(args []string, extra *model.CommandArgs) (*mod
 		license = config.E20License
 	}
 
-	if install.Version != "" || install.Image != "" {
-		err = validVersionOption(install.Version)
-		if err != nil {
-			return nil, true, err
-		}
+	err = validVersionOption(install.Version)
+	if err != nil {
+		return nil, true, err
+	}
 
-		var exists bool
+	var validTag bool
 
-		exists, err = p.dockerClient.ValidTag(install.Version, install.Image)
-		if err != nil {
-			p.API.LogError(errors.Wrapf(err, "unable to check if %s:%s exists", install.Image, install.Version).Error())
-		}
-		if !exists {
-			return nil, true, errors.Errorf("%s is not a valid docker tag for repository %s", install.Version, install.Image)
-		}
+	validTag, err = p.dockerClient.ValidTag(install.Version, install.Image)
+	if err != nil {
+		p.API.LogError(errors.Wrapf(err, "unable to check if %s:%s exists", install.Image, install.Version).Error())
+	}
+	if !validTag {
+		return nil, true, errors.Errorf("%s is not a valid docker tag for repository %s", install.Version, install.Image)
+	}
 
-		var digest string
-		digest, err = p.dockerClient.GetDigestForTag(install.Version, install.Image)
-		if err != nil {
-			return nil, false, errors.Wrapf(err, "failed to find a manifest digest for version %s", install.Version)
-		}
-		install.Version = digest
+	var digest string
+	digest, err = p.dockerClient.GetDigestForTag(install.Version, install.Image)
+	if err != nil {
+		return nil, false, errors.Wrapf(err, "failed to find a manifest digest for version %s", install.Version)
 	}
+	install.Version = digest
```

Comment:
Before we overwrite the version with the digest, we should store the original version value in `install.Tag` like we do on upgrades.

Reply:
Spoke with @IndushaS in chat. This was a mistake on my part and is already set above.